<launch>
  	<node name="map_to_sensor" pkg="cwru_nav" type="map_as_sensor" respawn="false" output="screen" >
	  <remap from="input_map" to="map"/>
	  <remap from="map_cloud" to="obstacle_map_cloud"/>
	  
		<param name="target_frame" value="base_link" />
		<param name="intensity" value="100.0"/>
		<param name="sensor_rate" value="1"/>
		<param name="channel_name" value="intensity"/>
		<param name="height" value="0.0"/>
	</node>
</launch>
